include/zephyr/dt-bindings/gpio/nordic-nrf-gpio.h

```diff
@@ -57,6 +62,8 @@
 #define NRF_GPIO_DRIVE_H0S1	(NRF_GPIO_DRIVE_H0 | NRF_GPIO_DRIVE_S1)
 /** High drive for '0' and '1' */
 #define NRF_GPIO_DRIVE_H0H1	(NRF_GPIO_DRIVE_H0 | NRF_GPIO_DRIVE_H1)
+/** Extra high drive for '0' and '1' */
+#define NRF_GPIO_DRIVE_E0E1	(NRF_GPIO_DRIVE_E0 | NRF_GPIO_DRIVE_E1)
```

Comment:
Why not include all other modes as well?
Full list is here:
https://github.com/zephyrproject-rtos/hal_nordic/blob/master/nrfx/hal/nrf_gpio.h#L197

Reply:
AFAIK `GPIO_PIN_CNF_DRIVE_E0E1` is the only mode currently supported by any nRF board, specifically the nRF5340, so I can't other modes can't be used at this time, and I have no board to test with :)

Reply:
See https://github.com/zephyrproject-rtos/hal_nordic/blob/ab5cb2e2faeb1edfad7a25286dcb513929ae55da/nrfx/mdk/nrf5340_application_bitfields.h#L8559
and https://github.com/zephyrproject-rtos/zephyr/blob/1ed04e7b49c5bec8e09f81480b7f95ae51b59b10/drivers/pinctrl/pinctrl_nrf.c#L16-L33